### PR TITLE
fix(theme-translations): add missing Vietnamese translations

### PR DESCRIPTION
**Pull request by namnguyenthanhwork**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [x] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

### CODE REVIEW

File: packages/docusaurus-theme-translations/locales/vi/theme-common.json [hunk 1]
```diff
@@ -3,32 +3,32 @@
...
-  "theme.admonition.info": "thông tin",
-  "theme.admonition.note": "ghi chú",
-  "theme.admonition.tip": "mẹo",
-  "theme.admonition.warning": "warning",
+  "theme.admonition.caution": "Cẩn thận",
+  "theme.admonition.danger": "Cảnh báo",
+  "theme.admonition.info": "Thông tin",
+  "theme.admonition.note": "Ghi chú",
+  "theme.admonition.tip": "Mẹo",
+  "theme.admonition.warning": "Cảnh báo",
```
Comment (slorber) [inline packages/docusaurus-theme-translations/locales/vi/theme-common.json:22]: Do we need to add uppercase here? 

The other languages do not have this and handle uppercasing with CSS afaik.

File: packages/docusaurus-theme-translations/locales/vi/theme-common.json [hunk 2]
```diff
@@ -61,10 +61,10 @@
   "theme.docs.sidebar.expandButtonTitle": "Mở rộng thanh bên",
   "theme.docs.sidebar.navAriaLabel": "Thanh điều hướng tài liệu",
   "theme.docs.sidebar.toggleSidebarButtonAriaLabel": "Đóng - mở thanh điều hướng",
-  "theme.docs.tagDocListPageTitle": "{nDocsTagged} với \"{tagName}\"",
+  "theme.docs.tagDocListPageTitle": "{nDocsTagged} được gắn thẻ \"{tagName}\"",
   "theme.docs.tagDocListPageTitle.nDocsTagged": "{count} tài liệu đã gắn thẻ",
   "theme.docs.versionBadge.label": "Phiên bản: {versionLabel}",
-  "theme.docs.versions.latestVersionLinkLabel": "phiên bản mới nhất",
+  "theme.docs.versions.latestVersionLinkLabel": "Phiên bản mới nhất",
```
Comment (slorber) [inline packages/docusaurus-theme-translations/locales/vi/theme-common.json:67]: Similarly I don't think we want uppercase here. The base language doesn't.

The final sentence would be:

> Để xem các cập nhật mới nhất, vui lòng xem phiên bản **Phiên** bản mới nhất


Code: 

```js
function LatestVersionSuggestionLabel({
  versionLabel,
  to,
  onClick,
}: {
  to: string;
  onClick: () => void;
  versionLabel: string;
}) {
  return (
    <Translate
      id="theme.docs.versions.latestVersionSuggestionLabel"
      description="The label used to tell the user to check the latest version"
      values={{
        versionLabel,
        latestVersionLink: (
          <b>
            <Link to={to} onClick={onClick}>
              <Translate
                id="theme.docs.versions.latestVersionLinkLabel"
                description="The label used for the latest version suggestion link label">
                latest version
              </Translate>
            </Link>
          </b>
        ),
      }}>
      {
        'For up-to-date documentation, see the {latestVersionLink} ({versionLabel}).'
      }
    </Translate>
  );
}
```

**Review by slorber — CHANGES_REQUESTED** [review]
LGTM but it's probably better if your translations have the same capitalization than the one from the base language (english)

File: packages/docusaurus-theme-translations/locales/vi/theme-common.json (hunk 1 above)
Comment (namnguyenthanhwork) [reply packages/docusaurus-theme-translations/locales/vi/theme-common.json:22]: If you using text-transform: uppercase; I think we don't need update them.

**Comment by slorber** [comment]
thanks 👍

**Pull request merged**